yarn.lock

```diff
@@ -2537,6 +2537,10 @@ hoek@2.x.x:
   version "2.16.3"
   resolved "https://registry.yarnpkg.com/hoek/-/hoek-2.16.3.tgz#20bb7403d3cea398e91dc4710a8ff1b8274a25ed"
 
+hoist-non-react-statics@^1.2.0:
```

Comment:
`mobx-react` depends on it.
```
yarn why hoist-non-react-statics
yarn why v0.17.10
[1/4] 🤔  Why do we have the module "hoist-non-react-statics"...?
[2/4] 🚚  Initialising dependency graph...
[3/4] 🔍  Finding dependency...
[4/4] 🚡  Calculating file sizes...
info This module exists because "mobx-react" depends on it.
info Disk size without dependencies: "72kB"
info Disk size with unique dependencies: "72kB"
info Disk size with transitive dependencies: "72kB"
info Amount of shared dependencies: 0
✨  Done in 0.70s.
```